nixos/modules/profiles/macos-builder.nix

```diff
@@ -23,17 +41,16 @@ in
       ];
       # swraid's default depends on stateVersion
       config.boot.swraid.enable = false;
-      options.boot.isContainer = lib.mkOption { default = false; internal = true; };
+      options.boot.isContainer = lib.mkOption {
+        default = false;
+        internal = true;
+      };
     }
   ];
 
+  options.virtualisation.description = "The maximum disk space allocated to the runner in megabytes";
```

Comment:
This can't be right. Declarations in `options` must be either individual options or nested attrsets of them. This is a string instead. It might not be accessed and therefore not be reported, but I'm not sure of that. Either way, this seems semantically wrong as well.

Reply:
Yes, not entirely sure where "diskSize" got missing here, tbh. But it's gone entirely now :)